src/Microsoft.Extensions.OData.Migration/README.md

```diff
@@ -0,0 +1,262 @@
+﻿# Introduction
+
+The OData Migration library provides ASP.NET Core 2.2+ OData V4 services with the capability of handling and responding to OData V3 requests.  An OData V4 service that uses the OData Migration Library may appear to an OData V3 client as the equivalent V3 service.  This may be useful to you if you have migrated your service from OData V3 to OData V4, but wish to bridge the gap between your newly migrated V4 service and your OData V3 clients.
+
+# Usage
```

Comment:
please prepare the similar md file and file an PR to 
[https://github.com/MicrosoftDocs/OData-docs/pulls](https://github.com/MicrosoftDocs/OData-docs/pulls)

Reply:
Will do